post_check.py

```diff
@@ -97,6 +106,25 @@ def main():
                             else:
                                 log_msg = "OTHER: " + clean_title
 
+                            curs.execute('''SELECT username, lastid, lastpost as "lastpost [timestamp]" FROM flair WHERE username=?''', (post.author.name,))
+
+                            row = curs.fetchone()
+
+                            # ensure that time of last post is > 24hrs
+                            if row is not None:
+                                if not row['lastid']:
+                                    lastid = ""
+                                else:
+                                    lastid = row['lastid']
+                                if row['lastpost']:
+                                    if (((((datetime.utcnow() - row['lastpost']).total_seconds() / 3600) < 24) and (((datetime.utcnow() - row['lastpost']).total_seconds() / 60) > 10)) and (lastid != "") and (post.id != lastid) and not post.approved_by):
```

Comment:
Done.  Originally left it hard coded because I wasn't aware of any sub that let people post more frequently than 24 hour.